roles/tomcat/defaults/main.yml

```diff
@@ -38,13 +38,13 @@ tomcat_cluster:
     expire_sessions_on_shutdown: "false"
     notify_listeners_on_replication: "true"
     receiver:
-      - address: ""
+      - address: "{{ tomcat_cluster_receiver[0].address }}"
```

Comment:
If there can be multiple receivers then `tomcat_cluster_receiver[0]` should be changed to `tomcat_cluster_receivers[0]`. Otherwise, `tomcat_cluster_receiver[0]` should be changed to `tomcat_cluster_receiver`. See also related question above about support for multiple receivers per group channel.